README.md

```diff
@@ -169,9 +169,12 @@ Some example SDK sizes from Bundlephobia:
 
 |SDK|Size|
 |----|----|
-|React|![React Bundle Size](https://img.shields.io/bundlephobia/minzip/@builder.io/sdk-react)|
+|React|![React Bundle Size](https://img.shields.io/bundlephobia/minzip/@builder.io/react)|
```

Comment:
should we put the package name along with the SDK? so that it is absolutely clear?

Reply:
Good call, done